lib/search.js

```diff
@@ -69,7 +69,9 @@ function search (args, silent, staleness, cb) {
     log.gauge.pulse('search')
     log.gauge.show({section: searchSection, logline: 'scanning ' + pkg.name})
     // Simply 'true' if the package matches search parameters.
-    var match = packageFilter(pkg, opts, searchexclude)
+    var match = packageFilter(pkg, opts, searchexclude, {
```

Comment:
Changing this to include the description flag is in keeping with the spirit of keeping the config out of the refactored search, but the cascading effects of this are pretty big. It's also weird that there are two options arguments here now.